spec/requests/v1/interventions/stars/delete_star_spec.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe 'PUT /v1/interventions/:id/star', type: :request do
```

Comment:
let's add more tests. For example: what happens when you pass not existing ID, this request is called by a participant, the researcher doesn't have access to the intervention